src/cephadm/cephadmlib/daemons/iscsi.py

```diff
@@ -119,22 +119,23 @@ def customize_container_binds(
     @staticmethod
     def get_version(ctx, container_id):
         # type: (CephadmContext, str) -> Optional[str]
-        version = None
-        out, err, code = call(
-            ctx,
-            [
-                ctx.container_engine.path,
-                'exec',
-                container_id,
-                '/usr/bin/python3',
-                '-c',
-                "import pkg_resources; print(pkg_resources.require('ceph_iscsi')[0].version)",
-            ],
-            verbosity=CallVerbosity.QUIET,
-        )
+        def python(s):
```

Comment:
Good idea, but I think that it should be a file level function, as there's very little need for it to be a closure. Something like:
```
def _python(engine: str, container_id: str, src: str) -> Tuple[str, str, int]:
    ...
```

Reply:
the whole point of creating this helper is to make the caller's life easier, and to reduce the number of parameters, the reason why i made it a nested function is that it is not supposed to be shared across other functions. and i wanted to make it very clear. but yes, it does not have to be a closure. 

i am moving it to a `staticmethod`, hope this helps.

Reply:
thank you John!